net/captivedetection/captivedetection.go

```diff
@@ -136,26 +136,31 @@ func interfaceNameDoesNotNeedCaptiveDetection(ifName string, goos string) bool {
 func (d *Detector) detectOnInterface(ctx context.Context, ifIndex int, endpoints []Endpoint) bool {
 	defer d.httpClient.CloseIdleConnections()
 
-	d.logf("[v2] %d available captive portal detection endpoints: %v", len(endpoints), endpoints)
+	use := max(len(endpoints), 5)
+	endpoints = endpoints[:use]
```

Comment:
Failure in CI on this line:
```
panic: runtime error: slice bounds out of range [:5] with capacity 2
```

Reply:
whoops: min, not max. good thing my work day is almost over.